package.json

```diff
@@ -68,6 +70,22 @@
     "slimerjs-edge": "0.10.0-pre-3",
     "wrench": "1.5.8"
   },
+  "eslintConfig": {
+    "extends": "openlayers",
+    "globals": {
+      "ArrayBuffer": false,
+      "Uint16Array": false,
+      "Uint32Array": false,
+      "Uint8Array": false,
+      "Uint8ClampedArray": false,
+      "ol": false,
```

Comment:
At this point I think we should only have implicit assignment with goog.provide, right?  When we get rid of that, ideally we'll be using es6 export (and perhaps have a build step that assigns to a global for a standalone build).  So I think we actually want to enforce that nobody tacks in arbitrary stuff to this right now.